hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/ReadableNodeStoreImpl.java

```diff
@@ -92,12 +93,18 @@ private Roster constructFromNodesState(@NonNull final ReadableKVState<EntityNumb
         for (final var it = nodesState.keys(); it.hasNext(); ) {
             final var nodeNumber = it.next();
             final var node = requireNonNull(nodesState.get(nodeNumber));
+            final var nodeEndpoints = node.gossipEndpoint();
+            // we want to swap the internal and external node endpoints
+            // so that the external one is at index 0
+            if (nodeEndpoints.size() == 2) {
+                Collections.swap(nodeEndpoints, 0, 1);
+            }
             if (!node.deleted()) {
                 final var entry = RosterEntry.newBuilder()
                         .nodeId(node.nodeId())
                         .weight(node.weight())
                         .gossipCaCertificate(node.gossipCaCertificate())
-                        .gossipEndpoint(node.gossipEndpoint())
+                        .gossipEndpoint(nodeEndpoints)
                         // (TSS-FUTURE) Enable node's TSS encryption key
```

Comment:
Can we remove the comments here about TSS encryption key.  That will never be enabled.

Reply:
Removed